apps/taxes/src/DashboardActions.tsx

```diff
@@ -0,0 +1,58 @@
+import { useAppBridge } from "@saleor/app-sdk/app-bridge";
+import { Button } from "@saleor/macaw-ui";
+
+/**
+ * This is example of using AppBridge, when App is mounted in Dashboard
+ * See more about AppBridge possibilities
+ * https://github.com/saleor/saleor-app-sdk/blob/main/docs/app-bridge.md
+ *
+ * You can safely remove this file!
```

Comment:
```suggestion
 * You can safely remove this file!
```
The comment doesn't lie! ;) 
The file should be removed